PLAT-145113: Fix to scroll by hover properly when there is no scrollbar (#970)
enactjs/sandstone: CHANGELOG.md

### Fixed

- `sandstone/Picker` value to not marquee when changing `title`

> - `sandstone/Scroller` and `sandstone/VirtualList` to scroll by hover when there is no scrollbar

Reviewer: ```suggestion
- `sandstone/Scroller` and `sandstone/VirtualList` to scroll by hover when scrollbar is hidden
```
